src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private int height;
+    private int width;
+    private String color = getColor();
+
+    public Rectangle(int height, int width, String color) {
+        this.height = height;
+        this.width = width;
+        this.color = color;
+    }
+
+    @Override
+    public double areaCalculator() {
+        return (height * width);
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure rectangle, area: " + areaCalculator()
+                + ", height " + height + ", width " + width
+                + ", color: " + color);
+    }
+}
```

Comment:
It seems that the class `Rectangle` is extending `Figure` which might be an abstract class. Ensure that you are using abstract classes and interfaces appropriately. If `Figure` is meant to define a contract for figures without providing any default behavior, it should be an interface instead.